specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/WafListPolicies.json

```diff
@@ -27,7 +27,25 @@
                 "fileUploadLimitInMb": 4000,
                 "requestBodyCheck": true,
                 "customBlockResponseStatusCode": 405,
-                "customBlockResponseBody": "SGVsbG8="
+                "customBlockResponseBody": "SGVsbG8=",
+                "logScrubbing": {
+                  "state": "Enabled",
+                  "scrubbingRules": [
+                    {
+                      "state": "Enabled",
+                      "matchVariable": "RequestArgNames",
+                      "selectorMatchOperator": "Equals",
+                      "selector": "test"
+                    },
+                    {
+                      "state": "Enabled",
+                      "matchVariable": "RequestIPAddress",
+                      "selectorMatchOperator": "EqualsAny",
+                      "selector": "*"
+                    }
+                  ]
+                }
+              },
```

Comment:
I think this is extra and needs to be removed.

Reply:
Addressed